copy/src/bin/main.rs

```diff
@@ -0,0 +1,198 @@
+// Copyright © 2019-2020 The Radicle Foundation <hello@radicle.foundation>
+//
+// This file is part of radicle-link, distributed under the GPLv3 with Radicle
+// Linking Exception. For full terms see the included LICENSE file.
+
+use std::{
+    convert::TryFrom as _,
+    path::{Path, PathBuf},
+};
+
+use anyhow::anyhow;
+use argh::FromArgs;
+
+use librad::{
+    git::{
+        identities::{self, local},
+        storage::Storage,
+        Urn,
+    },
+    git_ext::{OneLevel, RefLike},
+    internal::canonical::Cstring,
+    keys::{PublicKey, SecretKey},
+    paths::Paths,
+    peer::PeerId,
+    signer::{BoxedSigner, SomeSigner},
+};
+use radicle_keystore::{
+    crypto::{self, Pwhash},
+    pinentry::Prompt,
+    FileStorage,
+    Keystore,
+};
+
+const SECRET_KEY_FILE: &str = "librad.key";
```

Comment:
Awww, this guy again -- do we have this hardcoded in so many places that we'll need to stick to this filename forever?

Reply:
Hmmmm, we _could_ just look under the `keys` directory and pick the one that's there. If there's multiple then allow the user to specify which via a CLI param. So there are 3 scenarios:
1. No keys exist -- throw an error saying they need to initialise
2. There's a single file under `keys`, e.g. `keys/ringo.key`, and we use that
3. There are many files under `keys`, e.g. `keys/john.key`, `keys/paul.key`, `keys/george.key`, `keys/ringo.key`, and we throw an error saying PICK ONE

Reply:
Always the 0, 1, or many problem. I added my proposal above for now anyway. Maybe we should just fix it to a single key. Move fast and delete things :racing_car: